baybe/kernels/basic.py

```diff
@@ -13,6 +13,62 @@
 from baybe.utils.validation import finite_float
 
 
+@define(frozen=True)
+class CosineKernel(Kernel):
```

Comment:
Regarding the comments on this kernel: Could you elaborate a bit? I am honestly a bit surprised that this kenrel turns out to be problematic. Did you try to use it in some other occasions like e.g. in our examples? If we cannot properly use/test the kernels ourselves, I'd actually prefer to not include it for now.

Reply:
so far I have nomore details
tried a few numbers and priors, always same error of computational nature

Reply:
removed cosine kernel s discussed